src/main/java/com/volmit/iris/object/IrisEffect.java

```diff
@@ -228,7 +228,10 @@ public void apply(Player p,  GeneratorAccess g)
 		if(sound != null)
 		{
 			Location part = p.getLocation().clone().add(RNG.r.i(-soundDistance, soundDistance), RNG.r.i(-soundDistance, soundDistance), RNG.r.i(-soundDistance, soundDistance));
-			p.playSound(part, getSound(), (float) volume, (float) RNG.r.d(minPitch, maxPitch));
+
+			J.sr(() -> {
+				p.playSound(part, getSound(), (float) volume, (float) RNG.r.d(minPitch, maxPitch));
+			});
```

Comment:
playSound part should be ran sync, So I used scheduler to run this statement sync in an Async method

Reply:
I think `J.sr` is Sync Repeating. I think you want `J.s` there.

Reply:
J.sr was a runTask method, I changed it as you said. Thanks